ResearchKit/Common/ORKAnswerFormat.h

```diff
@@ -1185,6 +1188,19 @@ ORK_CLASS_AVAILABLE
 @interface ORKTextAnswerFormat : ORKAnswerFormat
 
 /**
+ Returns an initialized text answer format using the regular expression.
+ 
+ This method is one of the designated initializers.
+ 
+ @param expression                The regular expression used to valide the text.
+ @param invalidMessage            The text presented to the user when invalid input is received.
+ 
+ @return An initialized validated text answer format.
+ */
+- (instancetype)initWithValidationExpression:(NSString *)expression
+                              invalidMessage:(NSString *)invalidMessage NS_DESIGNATED_INITIALIZER;
+
+/**
  Returns an initialized text answer format using the specified maximum string length.
  
  This method is the designated initializer.
```

Comment:
Since you have two designated initializers now, update this comment to also say `This method is one of the designated initializer`.